pkg/driver/constants.go

```diff
@@ -82,6 +82,9 @@ const (
 	// BlockExpressKey increases the iops limit for io2 volumes to the block express limit
 	BlockExpressKey = "blockexpress"
 
+	// FSTypeKey configures the file system type that will be formatted during volume creation.
+	FSTypeKey = "csi.storage.k8s.io/fstype"
```

Comment:
If this isn't used in the driver itself, it should be in the e2e package.

Reply:
We should use it in the driver here: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/61c721b88933faec8fbb9c6610d5fc9d265ef5d2/pkg/driver/controller.go#L160

If all other parameter keys mentioned in our docs are in `constants.go` I think that FSTypeKey belongs there as well. Especially because if one day we do use this key in the driver for something more substantial, we would have a duplicated constant.